tests/components/usb/test_init.py

```diff
@@ -403,6 +403,7 @@ async def test_discovered_by_websocket_scan_limited_by_manufacturer_matcher(
             "domain": "test1",
             "vid": "3039",
             "pid": "3039",
+            "description": "*ConBee*",
             "manufacturer": "dresden elektronik ingenieurtechnik*",
```

Comment:
This doesn't seem to be testing the case insensitivity though..?

Reply:
I (lazily) extended an existing test. The description is `ConBee II` and the matcher is `*ConBee*`, which will fail with the old logic because `conbee ii` does not match `*ConBee*`. If you keep the unit test but revert the code change, the unit test will fail.

Reply:
Right, I see. I probably would've made either string obviously differently-cased from the other :)